erdpy_network/proxy_network_provider_test.py

```diff
@@ -106,3 +108,63 @@ def test_get_sc_invoking_tx(self):
         assert result.is_completed == True
         assert len(result.contract_results.items) > 0
         assert result.data == 'issue@54455354@54455354@03e8@00@63616e4d696e74@74727565@63616e4275726e@74727565@63616e4368616e67654f776e6572@74727565@63616e55706772616465@74727565'
+
+    def test_send_transactions(self):
```

Comment:
You can add a separate test for `send_transaction` (the singular version), though.

Reply:
Done!